MT/VR-MT-0006/VR-MT-0006.json

```diff
@@ -0,0 +1,94 @@
+{
+	"Identifier": "VR-MT-0006",
+	"Type": "Acceptance",
+	"Country": "MT",
+	"Version": "1.0.0",
+	"SchemaVersion": "1.0.0",
+	"Engine": "CERTLOGIC",
+	"EngineVersion": "0.7.5",
+	"CertificateType": "Vaccination",
+	"Description": [
+		{
+			"lang": "en",
+			"desc": "Verification Datetime must be less than 90 days after vaccination for 2nd dose"
+		}
+	],
+	"ValidFrom": "2022-01-16T00:00:00Z",
+	"ValidTo": "2030-06-01T00:00:00Z",
+	"AffectedFields": [
+		"v.0",
+		"v.0.dn",
+		"v.0.sd",
+		"v.0.dt"
+
+		
+	],
+	"Logic": {
+		"if": [
+			{
+				"var": "payload.v.0"
+			},
+			{
+				"if": [
+					{
+						"==": [
```

Comment:
This will not validate (after you've rename this file to `rule.json`): should be `"==="`.